src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -505,7 +505,7 @@ public virtual async Task<bool> UpdateTopologyAsync(UpdateTopologyParameters par
 
                 using (ContextPool.AllocateOperationContext(out JsonOperationContext context))
                 {
-                    var command = new GetDatabaseTopologyCommand(parameters.DebugTag, Conventions.SendApplicationIdentifier ? parameters.ApplicationIdentifier : null, _usePrivateUrls);
+                    var command = new GetDatabaseTopologyCommand(parameters.DebugTag, Conventions.SendApplicationIdentifier ? parameters.ApplicationIdentifier : null, _usePrivateUrls, Conventions._includePromotablesInTopology);
```

Comment:
create a private field like we do with `_usePrivateUrls`

Reply:
done